docs/_static/css/custom-theme.css

```diff
@@ -18,4 +80,155 @@
 
 .wy-nav-content {
     border-left: 2px #EEEEF1 solid;
-}
\ No newline at end of file
+    max-width: 1200px;
+}
+.caption {
+    border-top: 1px solid #eeeef1;
+}
+.caption-text {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:500;
+    font-size: 12px;
+    color: #666669;
+    line-height: 14px;
+    text-transform: capitalize;
+}
+
+.current {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 13px;
+    background: #fff;
+}
+
+.reference.internal {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:400;
+    font-size: 13px;
+    color: #555558;
+    line-height: 20px;
+    text-decoration: none;
```

Comment:
On the front page https://gro-intelligence.github.io/api-client/musenlin-gro-patch-1/index.html the table of contents don't look like links and have no hover effect to indicate that they are

Reply:
we don't show underline on the side menu. If you hover on the sidebar, you should see the texts turn to red.
<img width="216" alt="Screen Shot 2019-09-16 at 12 15 29 PM" src="https://user-images.githubusercontent.com/55099259/64974908-d0f7f980-d87b-11e9-8415-e14c222d0572.png">

Reply:
I'm not talking about the sidebar. Talking about on the main page here: https://gro-intelligence.github.io/api-client/musenlin-gro-patch-1/index.html

They don't have any hover effects. Tough to tell that they are links.